docs/js/guidlines.md

```diff
@@ -0,0 +1,60 @@
+# JavaScript Style Guidelines
+
+React Boilerplate subscribes to AirBnB's well established [JavaScript](https://github.com/airbnb/javascript) and [React](https://github.com/airbnb/javascript/tree/master/react) style guides with a few tweaks
+
+These tweaks are defined in the [`package.json`](../../tree/master/package.json) and some are explained throughout this document.
+
+While it's good practice to follow a common code style, the most important thing is that you and your team agree on a consistant coding style. The default style guides included here in React Boilerplate are highly recommended, but if your team would like to change some rules - feel free to do so!
+
+## Overrides
+
+### arrow-parens 
+
+eslint: [`arrow-parens`](http://eslint.org/docs/rules/arrow-parens)
+
+> Why? Not sure
```

Comment:
:P I thought that might be the case but wasn't sure!
More specifically, is it for consistency with 2+ arg functions? `( one ) =>` & `( one, two ) =>`?
I think that's a fair comment so I'll update this bit in the morning (it's getting late in Sydney!)